.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,61 @@
+name: Bug report
+description: Let us know about unexpected behaviour when using the CLI or Oxide Rust SDK
+body:
+  - type: checkboxes
+    id: preliminary
+    attributes:
+      label: Preliminary checks
+      description: Please verify the following.
+      options:
+        - label: I am using the latest version, or the latest version that corresponds to my Oxide installation.
+        - label: There is no open issue that reports the same problem.
+    validations:
+      required: true
```

Comment:
I'm reluctant to require customers experiencing an issue to go do additional work. Is it possible to include text without the checkboxes?

- Check that the version of the CLI or SDK matches that of the Oxide installation (we probably need to tell them how to do this and I'm not sure of the best way).
- See if there's already an issue filed for the behavior you're experiencing--there may already be a fix or work-around.